openfl-tutorials/interactive_api/MXNet_landmarks/envoy/sd_requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy
+pillow
+pynvml
```

Comment:
Looks like for now we do not use cuda monitor plugin, do we plan to use monitor plugin?

Reply:
Yes